frontend/src/components/Calendar/DayHeaders.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components'
+import React from 'react'
+
+const DayContainer = styled.div`
+  font-weight: 600;
+  font-size: 13px;
+  text-align: center;
+`
+
+export const DayHeaders = () => {
+  return (
+    <>
+      <DayContainer>Mon</DayContainer>
+      <DayContainer>Tue</DayContainer>
+      <DayContainer>Wed</DayContainer>
+      <DayContainer>Thu</DayContainer>
+      <DayContainer>Fri</DayContainer>
+      <DayContainer>Sat</DayContainer>
+      <DayContainer>Sun</DayContainer>
```

Comment:
I think there is a common file that has the days already, consider importing it and using map to create the day containers for each day

Reply:
The common file has the naming of the days in full (i.e. `Monday` vs `Mon`); 

would it be advisable to create a new array with abbreviated days in the common file?

Reply:
Fixed in [be93598](https://github.com/rhdevs/rhapp/pull/567/commits/be935981194bfb5664c81fa0d8fbd6d283d18978)